x/emissions/keeper/inference_synthesis/types.go

```diff
@@ -68,13 +68,12 @@ type SynthPalette struct {
 	ForecastByWorker                 map[Worker]*emissions.Forecast
 	ForecastImpliedInferenceByWorker map[Worker]*emissions.Inference
 	// Must respect the order of sister `forecasters` property
-	ForecasterRegrets    map[Worker]*StatefulRegret
-	AllInferersAreNew    bool
-	AllForecastersAreNew bool
-	AllWorkersAreNew     bool // Simple conjunction of the two above
-	NetworkCombinedLoss  Loss
-	Epsilon              alloraMath.Dec
-	FTolerance           alloraMath.Dec
-	PNorm                alloraMath.Dec
-	CNorm                alloraMath.Dec
+	ForecasterRegrets   map[Worker]*StatefulRegret
+	AllInferersAreNew   bool
+	SingleInfererNotNew Worker
```

Comment:
i get that it's more efficient this way to cache this value. it does feel cleaner to check this in the accumulator using the existing "noPriorRegret" prop. I can go either way